src/doc/trpl/concurrency.md

```diff
@@ -42,13 +42,18 @@ us enforce that it can't leave the current thread.
 ### `Sync`
 
 The second of these traits is called [`Sync`](../std/marker/trait.Sync.html).
-When a type `T` implements `Sync`, it indicates to the compiler that something
-of this type has no possibility of introducing memory unsafety when used from
-multiple threads concurrently.
-
-For example, sharing immutable data with an atomic reference count is
-threadsafe. Rust provides a type like this, `Arc<T>`, and it implements `Sync`,
-so it is safe to share between threads.
+When a type `T` implements `Sync`, it indicates that something of this type has
+no possibility of introducing memory unsafety when shared
+[immutably](mutability.html) between threads. This implies that types which
```

Comment:
From the [API docs](https://doc.rust-lang.org/std/marker/trait.Sync.html),

>The precise definition is: a type `T` is Sync if `&T` is thread-safe. In other words, there is no possibility of data races when passing `&T` references between threads.

`&T` can be called an "immutable reference", right?